app/services/SubmissionService.scala

```diff
@@ -52,7 +52,7 @@ class SubmissionService @Inject()(
     val timerContext = exportsMetrics.startTimer(submissionMetric)
     auditService.auditAllPagesUserInput(getCachedData(exportsDeclaration))
     for {
-      _ <- exportsConnector.create(exportsDeclaration).recover {
+      _ <- exportsConnector.createDeclaration(exportsDeclaration).recover {
```

Comment:
this is out of scope of this PR but here we would remove cache even connector would return an error

Reply:
I think the "Save and Resume" story will mean we no longer need the cache. I discussed with Steve (who is planning to pick this work up) that we can potentially remove the cache as part of it